lib/cube_browser/browse.py

```diff
@@ -0,0 +1,120 @@
+import glob
+import IPython.display
+import ipywidgets
+import iris
+import matplotlib.pyplot as plt
+
+import cube_browser
+
+# Clear output, such as autosave disable notification.
+IPython.display.clear_output()
+
+# Defines the file system path for input files.
+path = ipywidgets.Text(
+    description='Path:',
+    value=iris.sample_data_path('GloSea4'),
+)
+def handle_path(sender):
+    options = glob.glob('{}/*'.format(path.value))
```

Comment:
@marqh Isn't this just the same as `sender['owner'].value` or `sender['new']`? Is there a stylistic preference to explicitly referencing `path` ?